### ISSUE TITLE
[Question] JS dev bundle & lst zones

### ISSUE THREAD

**Issue by vicb** [angular/angular#1656]
I'm wondering why we have the following code in out gulpfile:

```js
gulp.task('bundle.js.dev.deps', ['bundle.js.dev'], function() {
  return bundler.modify(JS_DEV_DEPS.concat(['dist/build/angular2.dev.js']), 'angular2.dev.js')
      .pipe(insert.append('\nSystem.config({"paths":{"*":"*.js","angular2/*":"angular2/*"}});\n'))
      .pipe(insert.append('\nzone = zone.fork(Zone.longStackTraceZone);\n'))
      .pipe(gulp.dest('dist/bundle'));
});
```

Considering that we already use the lst in `vm_turn_zone.es6`:

```js
  _createInnerZone(zone, enableLongStackTrace) {
    var vmTurnZone = this;
    var errorHandling;

    if (enableLongStackTrace) {
      errorHandling = StringMapWrapper.merge(Zone.longStackTraceZone, {
        onError: function (e) {
          vmTurnZone._onError(this, e)
        }
      });
    } else {
      errorHandling = {
        onError: function (e) {
          vmTurnZone._onError(this, e)
        }
      };
    }
```

@rkirov any insights ?

**Comment by rkirov** [comment]
The important line is `var zone = new VmTurnZone({enableLongStackTrace: assertionsEnabled()});` which means assertions are already enabled in dev mode.

You can remove that line from the bundle code.

**Comment by mhevery** [comment]
Is it resolved? Can this be closed?

**Comment by vicb** [comment]
fixed in 8e1d53b5e9ac2e3e080efb46c29d6ed28eafca9c
